ginga/qtw/ImageViewQt.py

```diff
@@ -82,8 +92,12 @@ def paintEvent(self, event):
 
         # redraw the screen from backing pixmap
         painter = QPainter(self)
+        #painter = get_painter(self)
         rect = QtCore.QRect(x1, y1, width, height)
-        painter.drawPixmap(rect, self.pixmap, rect)
+        painter.drawPixmap(rect, pixmap, rect)
+        if DEBUG_MODE:
+            qimage = pixmap.toImage()
+            qimage.save('/tmp/final_image.png', format='png')
```

Comment:
Maybe use https://docs.python.org/3/library/tempfile.html instead of hardcoding the path?

Reply:
Will do. Or, may just remove the code. In any case I ought to make that a function call and fix it there.